### ISSUE TITLE
Update on pycocotools for windows

### ISSUE THREAD

**Issue by ai-fast-track** [airctic/icevision#258]
## 🚀 Feature

For Windows, we are currently installing the pycocotools from [philferriere ](https://github.com/philferriere/cocoapi)

We should use an updated version from [gautamchitnis ](https://github.com/gautamchitnis/cocoapi) using this command here below. This fork was created to update the philferriere clone with newer commits from original repo. :

`pip install git+https://github.com/gautamchitnis/cocoapi.git@cocodataset-master#subdirectory=PythonAPI`

This is the recommended method in [torchvision tutorial](https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html) : Check out the Windows note.

**Comment by oke-aditya** [comment]
I see no difference between the two repositories.

I just saw the commit history and only diffrence is update in readme.md and .gitignore file.
You can view 
1. New propose commits [here](https://github.com/gautamchitnis/cocoapi/commits/master)
2. Current Philferriere commits [here](https://github.com/philferriere/cocoapi/commits/master)

We can keep this issue open as "Update Pycoco Tools for Windows", maybe we can find a better option.

**Comment by oke-aditya** [comment]
So, we discussed over PyCoco tools and I guess this should be the final solution.

Let's keep PyCoco for Windows and Linux separate. So that in case either messes up we aren't struggling to deliver at least Linux builds.

1. We do not maintain PyCoco for Linux. We use the original one itself from COCO API.
2. For PyCoCo on windows. We try to maintain it. We have forked it to our organization.
3. Windows support will probably come in release 0.2.0, not in 0.1.0. So we can focus only on Linux right now and ensure that it works fine.
4. Windows support for release 0.1.0 is ABSENT. Not experimental. 0.2.0 will be experimental and hopefully, we can provide it.

To ship PyCoco tools for different distros as different requirements. We would need to use Poetry. Related to #267.
After we have these things in place. We can finally resolve Pycoco problem.

Also, we would have it experimental in 0.2.0. Beta in 0.3.0 and maybe in 0.4.0 stable support with Windows.

I like this policy from PyTorch. All the new features are in 3 zones. Experimental, Beta and Stable. So that people are clear what works and what doesn't.
